### PR TITLE
Extension methods

### PR DESCRIPTION
**Pull request by Cphrampus**
Renamed file and added git extension methods for common workflows

### CODE REVIEW

File: git-rml [hunk 1]
```diff
@@ -0,0 +1 @@
+git rm $@
```
Comment (Cphrampus) [inline git-rml]: gitdo

**Comment by ghost** [comment]
where the shebangs

how will i use this in fish >:(

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
 # git-functions
 A, hopefully growing, collection of scripts to help me use git better/easier/more efficently/etc.
+
+To ensure functionality, have this directory added to your path, or the scripts symlinked to a place that is.
+This will allow you to use gitdo itself, and the other scripts(git-*).
+
+To use the git-* scripts, as long as they are in your PATH, just type git <name of script after git->,
```
Comment (Cphrampus) [inline README.md]: angle brackets don't display literally in md

**Comment by Cphrampus** [comment]
The main file is shebanged, the other just call it, will fish not allow that without the additional shebanging?

**Comment by ghost** [comment]
nah its probably fine

**Comment by Cphrampus** [comment]
It won't hurt to add them, and it makes it more deterministic

**Pull request merged**